tests/e2e/login/purge.spec.js

```diff
@@ -6,7 +6,7 @@ const auth = require('../../auth')(),
 
 describe('Purging on login', () => {
 
-  const restrictedUserName = 'e2e_restricted',
+  const restrictedUserName = 'e2e_restricted' + new Date().getTime(),
```

Comment:
While it's tempting to do this (other e2e tests do) I don't think it's the right approach.

Instead, if it's failing because the user is not being deleted correctly we should fix that problem.

Otherwise we're just kicking the can down the road, and people's `_user` tables end up full of random users (as happens with the rest of the e2e tests right now).

Reply:
Removed and disabled this test per conversation below.